Apply this patch after running the following command in the sources directory:

gettextize --force

Tested with gettext 0.14.5.

Note: included libintl breaks more time than not, I would suggest to _not_ use
it at all, that's what this patch is supposed to do, too. For this reason after
running that command the files in intl subdirectory will be wiped, remove them
from cvs, too.
Index: xine-lib-20060606/configure.ac
===================================================================
--- xine-lib-20060606.orig/configure.ac
+++ xine-lib-20060606/configure.ac
@@ -159,17 +159,9 @@ dnl NLS Support
 dnl ---------------------------------------------
 
 dnl ALL_LINGUAS="cs da de el en@quot en@boldquot es et fr gl id it ja ko nl nn no pl pt pt_BR ru sl sv tr zh"
-ALL_LINGUAS="fr pt_BR de sk es pl cs it eu"
 AM_ICONV
-AM_GNU_GETTEXT(use-libtool, need-ngettext)
+AM_GNU_GETTEXT([external])
 AC_PROG_GMSGFMT_PLURAL
-AC_CONFIG_LINKS($nls_cv_header_libgt, $nls_cv_header_intl)
-AM_CONDITIONAL(INCLUDED_INTL, test x$USE_INCLUDED_LIBINTL = "xyes")
-if test x"$USE_INCLUDED_LIBINTL" = x"yes"; then
-   INTLDIR="-I\$(top_builddir)/intl -I\$(top_srcdir)/intl"
-fi
-AC_SUBST(INTLDIR)
-
 
 dnl ---------------------------------------------
 dnl Checks for typedefs, structures, and compiler characteristics.
@@ -2292,7 +2284,6 @@ doc/hackersguide/Makefile
 doc/faq/Makefile
 include/Makefile
 include/xine.h
-intl/Makefile
 lib/Makefile
 m4/Makefile
 misc/Makefile
Index: xine-lib-20060606/po/Makevars
===================================================================
--- /dev/null
+++ xine-lib-20060606/po/Makevars
@@ -0,0 +1,42 @@
+# Makefile variables for PO directory in any package using GNU gettext.
+
+# Usually the message domain is the same as the package name.
+DOMAIN = $(PACKAGE)
+
+# These two variables depend on the location of this directory.
+subdir = po
+top_builddir = ..
+
+# These options get passed to xgettext.
+XGETTEXT_OPTIONS = --keyword=_ --keyword=N_ --from-code=UTF-8
+
+# This is the copyright holder that gets inserted into the header of the
+# $(DOMAIN).pot file.  Set this to the copyright holder of the surrounding
+# package.  (Note that the msgstr strings, extracted from the package's
+# sources, belong to the copyright holder of the package.)  Translators are
+# expected to transfer the copyright for their translations to this person
+# or entity, or to disclaim their copyright.  The empty string stands for
+# the public domain; in this case the translators are expected to disclaim
+# their copyright.
+COPYRIGHT_HOLDER = Copyright (C) 2000-2006 the xine project
+
+# This is the email address or URL to which the translators shall report
+# bugs in the untranslated strings:
+# - Strings which are not entire sentences, see the maintainer guidelines
+#   in the GNU gettext documentation, section 'Preparing Strings'.
+# - Strings which use unclear terms or require additional context to be
+#   understood.
+# - Strings which make invalid assumptions about notation of date, time or
+#   money.
+# - Pluralisation problems.
+# - Incorrect English spelling.
+# - Incorrect formatting.
+# It can be your email address, or a mailing list address where translators
+# can write to without being subscribed, or the URL of a web page through
+# which the translators can contact you.
+MSGID_BUGS_ADDRESS =
+
+# This is the list of locale categories, beyond LC_MESSAGES, for which the
+# message catalogs shall be used.  It is usually empty.
+EXTRA_LOCALE_CATEGORIES =
+
Index: xine-lib-20060606/src/xine-engine/Makefile.am
===================================================================
--- xine-lib-20060606.orig/src/xine-engine/Makefile.am
+++ xine-lib-20060606/src/xine-engine/Makefile.am
@@ -24,12 +24,9 @@ libxine_la_SOURCES = xine.c metronom.c c
 # FIXME: these are currently unused:
 EXTRA_DIST = lrb.c lrb.h accel_xvmc.h
 
-if INCLUDED_INTL
-INCLUDED_INTLLIBS = @INTLLIBS@
-endif
-libxine_la_DEPENDENCIES = $(INCLUDED_INTLLIBS) $(XINEUTILS_LIB) \
+libxine_la_DEPENDENCIES = $(XINEUTILS_LIB) \
                           $(pthread_dep) $(LIBXINEPOSIX)
-libxine_la_LIBADD = $(THREAD_LIBS) $(DYNAMIC_LD_LIBS) @INTLLIBS@ $(ZLIB_LIBS) \
+libxine_la_LIBADD = $(THREAD_LIBS) $(DYNAMIC_LD_LIBS) $(LTLIBINTL) $(ZLIB_LIBS) \
 		    -lm $(XINEUTILS_LIB) $(LIBICONV) $(FT2_LIBS) \
                     $(LIBXINEPOSIX) $(RT_LIBS)
 
@@ -54,8 +51,5 @@ uninstall-local:
 	rm -f $(DEF_FILE)
 endif
 
-@INCLUDED_INTL_TRUE@@INTLLIBS@:
-@INCLUDED_INTL_TRUE@	@cd $(top_builddir)/intl && $(MAKE) libintl.la
-
 $(XINEUTILS_LIB):
 	$(MAKE) -C $(top_builddir)/src/xine-utils libxineutils.la
